app/src/androidTest/java/com/github/sdp_begreen/begreen/fragments/SendPostFragmentTest.kt

```diff
@@ -97,9 +98,7 @@ class SendPostFragmentTest {
     @Before
     fun setup() {
         fragmentScenario = launchFragmentInContainer(
```

Comment:
Here is how I would write the code to omit the `param_test` argument in `SendPostFragment`.
In reference with what I said in the `SendPostFragment` file.

```suggestion
        fragmentScenario = launchFragmentInContainer { CameraContainer.newInstance() }
            .onFragment {
                it.parentFragmentManager.commit {
                    setReorderingAllowed(true)
                    replace(R.id.mainCameraFragmentContainer, SendPostFragment.newInstance(user.id) )
                }
            }
```

Reply:
Thank you!